assignment7.txt

```diff
@@ -0,0 +1,22 @@
+
+
+1. ++*p increments what the pointer points to whereas the *p++ actually increases the value of the pointer. *++p is also increases what the pointer points to.
+
+
+2. Oh the left is guaanteed because its higher in the order of operations.
+
+
+3. pointers allow easier use with arrays. You can go through the pointer without the for loop and it allows easier access outside of the main function. 
+
+
+4.1 char array 
+4.2 Invalid becaus xyz in quotes is not an arry
```

Comment:
This is actually valid! "xyz" is just an array of characters, so the "xyz"[1] is just accessing "y". Then you just subtract 'y' from it to get 0. This is because, if you can recall, a char is just a value mapped to a character!